packages/nbdime/src/common/mergeview.ts

```diff
@@ -123,14 +123,26 @@ const mergeViewDecorationDict: MergeViewDecorationDict = {
   mixedMerge: Private.buildEditorDecorationDict('m', 'mixed'),
 };
 
+/**
+ * Additional theme element
+ */
+const baseTheme = EditorView.baseTheme({
+  '.cm-collapsedLines': {
+    padding: '5px 5px 5px 10px',
+    cursor: 'pointer',
+    color: 'var(--jp-ui-font-color2)',
+    outline: 'solid var(--jp-border-width) var(--jp-border-color1)',
+  },
+});
+
```

Comment:
```suggestion
const baseTheme = EditorView.baseTheme({
    cursor: 'pointer',
    outline: 'solid var(--jp-border-width) var(--jp-border-color1)',
    color: 'var(--jp-ui-font-color1)',
    backgroundColor: 'var(--jp-layout-color2)',
    border: '1px solid var(--jp-border-color1)',
    fontSize: '90%',
    padding:'0 3px',
    borderRadius: '4px'
  },
});

```
The suggestion proposed here is the following one or the rendering of the collapsers:
![Screenshot from 2023-10-03 16-51-58](https://github.com/jupyter/nbdime/assets/99649086/cf32ce07-7c94-4817-bbeb-5a12f777cd95)